docs/getting-started.md

```diff
@@ -215,3 +216,16 @@ cd DfE.FindInformationAcademiesTrusts
 npm run lint ## for a list of issues
 npm run lint:fix ## to scan and fix issues
 ```
+
+## Application Insights
+
+App insights is the platform we are using for measuring telemetry. By default app insights is on in dev and test environments and off when running locally or building in a pipeline.
+
+App insights can be enabled locally by including the conncection string in your secrets file. The format should be `"APPLICATIONINSIGHTS_CONNECTION_STRING": "<Connection string here"`
+You can copy the connection string from Azure. Always use the development environment when testing locally.
```

Comment:
Where in Azure? Could this be a bit more specific?

You can copy the connection string from the relevant...(application insights instance in Azure Portal - or similar)